server/src/main.ml

```diff
@@ -33,7 +33,11 @@ let () =
   Logs.set_reporter (combine logfile err_std);
   Logs.set_level (Some Logs.Info);
   Logs.info (fun m -> m "log file created: '%s'" logfile_path);
-  Catala_utils.Global.enforce_options ~message_format:Lsp () |> ignore
+  Catala_utils.Global.enforce_options
+  (* FIXME: this still prints warnings on stderr due to [Message] hard-wiring
```

Comment:
Indeed, Message.ml should let you configure at the global level where you want the output to be (stderr, stdout, file, etc).